app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    parts: list[str] = command.split()
+    if len(parts) != 3 or parts[0] != "mv":
+        raise ValueError("Wrong command. Format: mv source destination")
+
+    source: str = parts[1]
+    destination: str = parts[2]
+
+    if not os.path.isfile(source):
+        raise FileNotFoundError(f"File {source} not found.")
+
+    if destination.endswith("/"):
+        os.makedirs(destination, exist_ok=True)
+        destination = os.path.join(destination, os.path.basename(source))
```

Comment:
When the destination ends with a slash, it is assumed to be a directory. The code correctly creates the directory if it doesn't exist and appends the source file name to the destination path. Ensure that this behavior is intended and documented for users of the function.